src/mbgl/annotation/annotation_manager.cpp

```diff
@@ -167,13 +159,26 @@ void AnnotationManager::updateStyle(Style& style) {
         shape.second->updateStyle(style);
     }
 
+    for (const auto& image : images) {
+        if (!style.getImage(image.first)) {
+            style.addImage(image.first, std::make_unique<style::Image>(image.second));
```

Comment:
For clarity, I'd rather copy; it's cheap, because all that's being copied is an `Immutable`, which is a shared pointer.